doc/security/vulnerabilities.rst

```diff
@@ -821,6 +821,174 @@ Integer Underflow in 6LoWPAN IPHC Header Uncompression
 
 - This issue has not been fixed.
 
+CVE-2021-3430
+-------------
+
+Assertion reachable with repeated LL_CONNECTION_PARAM_REQ.
+
+This has been fixed in main for v2.6.0
+
+- `CVE-2021-3430 <http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-3430>`_
+
+- `Zephyr project bug tracker GHSA-46h3-hjcq-2jjr
+  <https://github.com/zephyrproject-rtos/zephyr/security/advisories/GHSA-46h3-hjcq-2jjr>`_
+
+- `PR 33272 fix for main
```

Comment:
if you do :github:\`33272\` it will create the link automatically.. :)

Reply:
Good to know, will start using it.